integreat_cms/cms/templates/chat/_chat_messages.html

```diff
@@ -22,7 +22,7 @@
             <i icon-name="trash-2"></i>
         </button>
         {# djlint:on #}
-        <span class="text-gray-600 float-right">{{ message.sent_datetime }}</span>
+        <span class="text-gray-600 float-right">{{ message.sent_datetime|date:"DATETIME_FORMAT" }}</span>
```

Comment:
```suggestion
        <span class="text-gray-600 float-right">{{ message.sent_datetime }}</span>
```
This is superfluous. You only need to specify the format if it differs from the default format. But since `DATETIME_FORMAT` is the default, you don't need this here. You only need it in places where `SHORT_DATE_FORMAT` should be used instead of the default format.